addons/project_timesheet_holidays/models/hr_holidays.py

```diff
@@ -8,46 +8,8 @@
 class HolidaysType(models.Model):
     _inherit = "hr.leave.type"
 
-    timesheet_generate = fields.Boolean(
-        'Generate Timesheets', compute='_compute_timesheet_generate', store=True, readonly=False,
+    timesheet_generate = fields.Boolean('Generate Timesheets', readonly=False, store=True,
```

Comment:
Is this field still needed ? From what i understand from the spec of the task, every computation and timesheet generation is done purely from the configuration done in the settings since the user is no longer able to set a value for this field from the view form. Am i missing something ? :thinking: